vllm/executor/ray_gpu_executor.py

```diff
@@ -189,6 +190,20 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
                           max_concurrent_workers=self.parallel_config.
                           max_parallel_loading_workers)
 
+        self.tp_driver_workers = []
+        self.tp_parallel_workers = []
```

Comment:
Add comments and typing here.

Reply:
I think the naming here could also be improved. `tp_parallel_workers` is a bit confusing; it sounds like it could just be the workers in the TP group with the driver. Maybe `non_driver_workers`?

It would be great if you can also note in the comment what each list index corresponds to.

Reply:
Done both. I didn't include the list index as we never actually index these lists, rather we just use them in map style operations.